source/guide_pelican.rst

```diff
@@ -0,0 +1,182 @@
+.. highlight:: console
+
+.. author:: Karsten Brusch <https://k11h.de>
+
+.. tag:: lang-python
+.. tag:: web
+.. tag:: blog
+
+
+.. sidebar:: Logo
+
+  .. image:: _static/images/pelican.png
+      :align: center
+
+#########
+Pelican
+#########
+
+.. tag_list::
+
+Pelican_ is a static site generator implemented in Python_ that combines Jinja templates with content written in Markdown or reStructuredText to produce websites.
+The most prominent example is probably `kernel.org <https://www.kernel.org/pelican.html>`_
+It is possible to `import <https://docs.getpelican.com/en/latest/importer.html>`_ an existing sites from Wordpress, Tumblr, Blogger and RSS/Atom feeds.
+Pelican's source code is available on `GitHub <https://github.com/getpelican/pelican>`_ and it is an implementation of the static site generators concept.
+
+Pelican was released for the first time in 2010 by Alexis Métaireau.
+
+----
+
+License
+=======
+
+Pelican is free and open source software licensed under `AGPL 3 <https://www.gnu.org/licenses/agpl-3.0.en.html>`_.
+
+Prerequisites
+=============
+
+Pelican currently runs best on Python 3.6+; earlier versions are not supported. 
+This version comes preinstalled on your uberspace.
+
+.. note:: If you want to use your own website domain, refer to the uberspace manual for :manual:`domains <web-domains>`
+
+Installation
+============
+
+Pelican is installed simply by using pip3
+
+::
+
+ [isabell@stardust isabell]$ pip3 install --upgrade --user "pelican[markdown]"
+
+.. note:: If you prefer to install the latest bleeding-edge version of Pelican rather than a stable release, use the following command:
+
+  ::
+
+    pip3 install --upgrade --user -e "git+https://github.com/getpelican/pelican.git#egg=pelican"
+
+
+Generate your website
+=====================
+
+**Step 1:** 
```

Comment:
Please give your steps clear names instead of numbers and make them proper headings. Numbered steps are very cumbersome when the guide is edited later.
```suggestion
Initialize
----------
```